css/footer.css

```diff
@@ -0,0 +1,50 @@
+/* footer */
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-top: 1px solid #ccc;
+  width: 50%;
+  margin: 6% auto;
+}
+
+.footer-title {
+  margin: 30px 0;
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: row;
+}
+
+.twitter,
+.facebook,
+.instagram {
+  border: 1px solid #ccc;
+  border-radius: 100%;
+  width: 35px;
+  height: 35px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 10px;
+}
+
+.twitter > img,
+.facebook > img,
+.instagram > img {
+  height: 20px;
+}
```

Comment:
* You can define one CSS classes and apply to different HTML elements instead of making different classes.